VisualStudio.gitignore

```diff
@@ -80,6 +80,8 @@ ipch/
 *.opensdf
 *.sdf
 *.cachefile
+*.VC.db
+*.VC.VC.opendb
```

Comment:
If we add `*.VC.opendb` we won't need to add `*.VC.VC.opendb`. @anurse any objection to removing `*.opendb`?